platforms/jvm/language-jvm/src/main/java/org/gradle/api/tasks/compile/DebugOptions.java

```diff
@@ -43,6 +43,8 @@ public class DebugOptions extends AbstractOptions {
      *     <dd>Local variable debugging information
      * </dl>
      *
+     * Also instead of those values {@code none} can be used.
```

Comment:
Why note include `none` in the above list?
Applies to the `setDebugLevel` javadoc as well

Reply:
My approach was that it is either a combination of `source`, `line`, `vars`, or just `none`. (And of course nothing at all, according to javadoc)

Reply:
I see. 

How does the behavior differ passing in `null` or `"none"`? 

I think we should only recommend one of these approaches instead of both

Reply:
Actually I think `none` would be preferred. I believe if `null` is used then `-g` is actually applied.
Also [javac `-g` manual](https://docs.oracle.com/en/java/javase/17/docs/specs/man/javac.html#option-g) inidcates that on the contrary `-g` enables all.

Reply:
Also maybe the same effect can be applied via `options.isDebug = true` 🤔